### DISCUSSION TITLE
Usage Endpoint: When is data available?

### DISCUSSION THREAD

**Discussion by melvanderwal** [amberelectric/public-api#133] [Q&A] [answered]
Hi all.

I've got a scheduled Node-RED flow that is pulling yesterday's data from the `sites/xyz/usage` endpoint and pushing it to InfluxDB.  It ran at 1 am last night and didn't get any data.  At what time should yesterday's data be available?

Thanks,
Mel

**Comment by madpilot** [comment, marked as answer]
It's not defined. Around 4am is a good bet, but it can be delayed as late as the afternoon, it depends on what is happening in the market.

**Reply by melvanderwal** [reply]
Just checking to make sure that I haven't borked a rate limit... usage API hasn't been providing results for a couple of days?  I'm not seeing usage data more recent than Tuesday in my flow or in the Amber Android app. Excited to see see what the result of my HA automation is for discharging my battery during yesterday's price spikes.

**Reply by purcell-lab** [reply]
>  I'm not seeing usage data more recent than Tuesday in my flow or in the Amber Android app. 

I'm also not seeing any results since Tuesday, sometimes  it lags a couple of days, but then generally catches up. Nothing to worry about.

**Reply by madpilot** [reply]
The problem is up stream of us - our data provider hasn't been receiving usage correctly. They are working on it.

**Reply by icenov** [reply]
Usage data still delayed? No update here since Tuesday - NSW 2055

**Reply by madpilot** [reply]
It's been escalated with our data provider. I don't know any more at this stage unfortunately

**Reply by madpilot** [reply]
Looks like mid Monday is the ETA

**Reply by icenov** [reply]
Any update? Almost 7 days without usage - very difficult to tune storage without knowing....

**Reply by purcell-lab** [reply]
> Any update? Almost 7 days without usage - very difficult to tune storage without knowing....

I received an update notice via the app and the data is starting to flow through now.

**Reply by icenov** [reply]
No update here yet...
